website/docs/highlights/aggregation.md

```diff
@@ -0,0 +1,28 @@
+---
+id: aggregation
+title: Aggregation
```

Comment:
Is this really about aggregation, or traversal? Actually child-to-parent aggregation is where H3 is weaker than many of its competitors.

Reply:
May need to be renamed, but roughly this is the "aggregation use case" rather than highlighting aggregation within the library. I'm not sure if "traversal" would be meaningful to a reader?